### DISCUSSION TITLE
CC:T 1.96.0 marked as 1.16.4 in maven

### DISCUSSION THREAD

**Discussion by Mathilde411** [cc-tweaked/CC-Tweaked#835] [Q&A] [answered]
Hello ! While searching for CC:T newest version I found issue https://github.com/SquidDev-CC/CC-Tweaked/issues/808
I was wondering if it was normal that version 1.96.0 is still referenced as 1.16.4 on maven ( https://squiddev.cc/maven/org/squiddev/cc-tweaked-1.16.4/1.96.0/ ) whereas it is mared as 1.16.5 in the gradle.properties.
Thanks :)

**Comment by SquidDev** [comment, marked as answer]
The bump to 1.16.5 happened after the 1.96.0 release, hence us still using 1.16.4 on maven.

**Reply by Mathilde411** [reply]
Okay, so the current version is not published yet, thanks ^^
